### ISSUE TITLE
Add a `--footer` flag

### ISSUE THREAD

**Issue by Delapouite** [ogham/exa#1159]
Hello

exa currently provides a `--header` flags providing the names of the column.
A `--footer` flag would be greatly appreciated to show information like : number of entries in the list, sum of the sizes…

Thanks

**Comment by eggbean** [comment]
I don't think this is a good idea when there are other tools that can do this. Exa is an ls replacement and it already does more than ls, but those are useful enhancements but already diverging from the [Unix tool philosophy](https://en.wikipedia.org/wiki/Unix_philosophy?oldformat=true#Do_One_Thing_and_Do_It_Well). For something used possibly over a hundred times a day, speed should not be sacrificed for anything.

**Comment by daviessm** [comment]
I actually quite like this idea, it's similar to how `dir` works in Windows. Exa is already finding each file size so adding them together and counting all the files listed would have basically zero speed penalty.

@Delapouite what information exactly do you think would be useful in a footer?
